core/src/test/java/org/elasticsearch/cluster/routing/allocation/decider/DiskThresholdDeciderTests.java

```diff
@@ -916,4 +915,25 @@ public boolean apply(ShardRouting input) {
                 rn.shardsWithState(RELOCATING),
                 rn.shardsWithState(STARTED));
     }
+
+    public static final Map<ShardRouting, String> DEV_NULL_MAP = new SingelValueMap("/dev/null");
+
+    private static final class SingelValueMap extends AbstractMap<ShardRouting, String> {
```

Comment:
`Collections.singletonMap()` holds one key - value pair. this map returns the same value not matter of the key